src/test/store/profile-view.test.js

```diff
@@ -1552,6 +1555,135 @@ describe('actions/ProfileView', function () {
       expect(screenshots.length).toEqual(endIndex - startIndex + 1);
     });
   });
+
+  describe('selectTrackFromTid', function () {
```

Comment:
Thanks for the tests!

As we discussed, it would be good to move this to `MarkerTable.test.js` so that we can test from the user action to the result for this action.